packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -215,8 +221,17 @@ function useMemo<T>(
   return value;
 }
 
-function useListener() {
-  throw new Error('TODO: not yet implemented');
+function useListener(
+  responder: ReactEventResponder<any, any>,
+  hookProps: ?Object,
+): void {
+  const listenerProps = hookProps || emptyObject;
+  // Don't put the actual event responder object in, just its displayName
+  const value = {
+    responder: responder.displayName,
```

Comment:
Should we set a default/fallback name (we do elsewhere for event components)

Reply:
Event responders should always have a mandatory part of the `unstable_createResponder` API. I can add that though!

Reply:
Ah! Okay well if we're _guaranteed_ to have a name then no need. I just wasn't sure, based on the precedent in the renderer interface. 😄

Reply:
I added it anyway, to be safe!